src/NuGet.Core/NuGet.Build.Tasks/BuildTasksUtility.cs

```diff
@@ -553,7 +553,7 @@ private static IEnumerable<Packaging.PackageReference> GetInstalledPackageRefere
                        projectConfigFilePath,
                        ex.Message);
 
-                    log.LogError(message);
+                    throw new XmlException(message);
```

Comment:
[Line 402](https://github.com/NuGet/NuGet.Client/pull/3207/files#diff-dc13cfd7bd242d1b20b434573a3f344aR402) will now throw on the first PackageSpec that fails. Previously this was logging and continuing. Intentional?

Reply:
> [Line 402](https://github.com/NuGet/NuGet.Client/pull/3207/files#diff-dc13cfd7bd242d1b20b434573a3f344aR402) will now throw on the first PackageSpec that fails. Previously this was logging and continuing. Intentional?

It was intentional. When I executed MSBuild /t:restore command for packages.config scenario noticed that there were 2 error messages about the same exception. 

![image](https://user-images.githubusercontent.com/52756182/73296202-15968880-41be-11ea-8dac-d5fd5654332c.png)

Here are the 2 callers (from top of the call-stack) where exceptions are logged. Hence I modified the source code to throw the exception to the caller with the new error message instead of logging to avoid duplication in the logs.
- https://github.com/NuGet/NuGet.Client/blob/0ba1a97052d9ba3f581bab5ab173270dac6681f0/src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs#L120

- https://github.com/NuGet/NuGet.Client/blob/0ba1a97052d9ba3f581bab5ab173270dac6681f0/src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs#L124